src/ucs/sys/topo.h

```diff
@@ -24,6 +24,17 @@ typedef struct ucs_sys_bus_id {
 } ucs_sys_bus_id_t;
 
 
+/**
+ * @brief System Device type enumeration
+ * Using a bit mask to allow the possibility of a device that may have memory 
+ * and can also move data
+ */
+typedef enum ucm_sys_dev_type {
+	UCS_SYS_DEV_UNKNOWN = 0,          /* Default */
```

Comment:
How is this supposed to be handled?

Reply:
UCS_SYS_DEV_UNKNOWN signifies that the ucs_system_device_t entry is invalid and shouldn't be used. The intention was to initialize a list of entries with UCS_SYS_DEV_UNKNOWN and ignore all entries which aren't marked UCS_SYS_DEV_MEM or UCS_SYS_DEV_NET

Reply:
looks redundant for me, you can just memset(0) the list and use entries which match valid device flags

Reply:
I agree as well. I'll discard this and keep just MEM and NET enums.